GitUpKit/Views/GIMapViewController+Operations.m

```diff
@@ -622,8 +622,8 @@ - (void)mergeCommitOrBranch:(id)commitOrBranch intoLocalBranch:(GCHistoryLocalBr
                                 [self.repository setUndoActionName:[NSString stringWithFormat:NSLocalizedString(@"Merge Commit Into \"%@\" Branch", nil), branch.name]];
                               }
                               if ([self.repository performReferenceTransformWithReason:(isBranch ? @"merge_branch" : @"merge_commit")
-                                                   argument:(isBranch ? [commitOrBranch name] : [commitOrBranch SHA1])
-                                                   error:&error
+                                                                              argument:(isBranch ? [commitOrBranch name] : [commitOrBranch SHA1])
+                                                                              error:&error
```

Comment:
Confused by this style. Isn't it usually colon aligned if it's in this style? Seems inconsistent here as all the other params are colon aligned except `error`

Reply:
Confused as well. Might be related it being in a `()`. Will look

Reply:
Seemingly it's getting confused by the prior line's `(isBranch ? [commitOrBranch name] : [commitOrBranch SHA1])`